### PR TITLE
Fix for #73

### PR DESCRIPTION
**Pull request by guyboertje**
added spec
fix typo
namespace Dummy* classes
better name for module LogstashTest

### CODE REVIEW

File: spec/support/logstash_test.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 require "stud/temporary"
-module LogStashTest
+
+
+# namespace the Dummy* classes, they are reused names
+# use a more specific module name to prevent clashes
+module BeatsInputTest
```
Comment (ph) [inline spec/support/logstash_test.rb]: I am +1 with that change

**Comment by ph** [comment]
LGTM, I have checked the failings test, Its a timing issue with the LSF integration test, the fun of dealing with external software.

**Comment by ph** [comment]
Go ahead and to merge and publish

File: spec/support/logstash_test.rb (hunk 1 above)
Comment (guyboertje) [inline spec/support/logstash_test.rb]: Great

**Pull request merged**